src/System.Device.Gpio/System/Device/Gpio/IGpioController.cs

```diff
@@ -6,6 +6,7 @@ namespace System.Device.Gpio
 {
     public interface IGpioController : IDisposable
     {
+		void SetDefaultGpioChip(int devGpioChip);
```

Comment:
I'm wondering if instead we should perhaps have i.e. (or just a constructor argument):
`LibGpiodController.Create(chip = 1)` - this way we don't need to change the interface and you can still work correctly in your scenario

Reply:
Agree better to have constructor argument

Reply:
This approach appears good to me

Reply:
oh i misread @krwq's comment, I actually meant pass to the LibGpiodDriver constructor not to the controller. @krwq might meant the same too.  Our Drivers are public, so you can create an instance and set to the controller directly 
`GpioController controller = new GpioController(PinNumberingScheme.Logical , new LibGpiodDriver(chip=1))`
Don't need to set chip number controller level https://github.com/dotnet/iot/pull/346/files#diff-7fe07ed0a542d5e8abc343a8cd469bc1R9

Reply:
I edited the description of PR with the new example of use